tests/coordinate_test.cpp

```diff
@@ -1,9 +1,12 @@
 #include "catch/catch.hpp"
 
-#include "coordinates.h"
-#include "coordinate_conversions.h"
+#include <string>
+#include <tuple>
+
 #include "cata_generators.h"
-#include "stringmaker.h"
```

Comment:
There are a few places where you've removed `stringmaker.h`.  This specializes some templates to customize Catch2 output and should remain.  I don't think there's any way for IWYU to know this so I guess it will have to get a pragma everywhere we include it.

Don't know whether you want to go to the trouble of reverting this now, but it's something to bear in mind for the future if you run IWYU again or see `{?}` printed out in test results.